.github/workflows/pr-tests-stack-public.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       max-parallel: 99
       matrix:
         # issues with macos 14 arm https://github.com/crazy-max/ghaction-setup-docker/pull/53
-        os: [ubuntu-latest, macos-13, windows]
```

Comment:
We used to use windows for self hosted runners, currently we do not have the same.Good catch